Rakefile

```diff
@@ -53,55 +53,17 @@ task :clean do
 end
 
 SOLIDUS_GEM_NAMES = %w[core api backend sample]
+GEM_TASKS_NAME = %w[build install release]
 
-namespace :gem do
-  def version
-    require_relative 'core/lib/spree/core/version'
-    Spree.solidus_version
-  end
-
-  def for_each_gem
+GEM_TASKS_NAME.each do |task_name|
+  desc "Run rake gem:#{task} for each Solidus gem"
+  task task_name do
     SOLIDUS_GEM_NAMES.each do |gem_name|
       print_title(gem_name)
-      yield "pkg/solidus_#{gem_name}-#{version}.gem"
-    end
-    print_title
-    yield "pkg/solidus-#{version}.gem"
-  end
-
-  desc "Build all solidus gems"
-  task :build do
-    pkgdir = File.expand_path('pkg', __dir__)
-    FileUtils.mkdir_p pkgdir
-
-    SOLIDUS_GEM_NAMES.each do |gem_name|
       Dir.chdir(gem_name) do
-        sh "gem build solidus_#{gem_name}.gemspec"
-        mv "solidus_#{gem_name}-#{version}.gem", pkgdir
+        sh "rake #{task_name}"
       end
     end
-
-    print_title
-    sh "gem build solidus.gemspec"
-    mv "solidus-#{version}.gem", pkgdir
-  end
-
-  desc "Install all solidus gems"
-  task install: :build do
-    for_each_gem do |gem_path|
-      Bundler.with_clean_env do
-        sh "gem install #{gem_path}"
-      end
-    end
-  end
-
-  desc "Release all gems to rubygems"
```

Comment:
I was under the impression that all of this code was not used at all by `gem-release`. Instead, this was the previous system used to release (see https://github.com/solidusio/solidus/pull/83). I think the subgems release was managed by the `recurse` option set in the `gem-release` config.

That said, I'm in favor of removing the dependency and simplifying the process but I have two concerns:

1. I suspect with the proposed changes we are not releasing the main `solidus` gem but only the subgems.
2. Having two different ways of releasing for new/older versions might create complications. Is there a way to uniform the two? Both having the same interface or backporting is ok for me, but if you have other ideas, they are welcome.

Reply:
Thanks, yes the recurse option is what got in the way with solidus_admin.

1. Good catch! Fixed 👍 
2. Added back ports labels for all supported versions